README.md

```diff
@@ -1,38 +1,40 @@
 
 ## Learning Goals
-- Extend code provided
 - Practice using events
 - Pass custom information to an event handler function
 - Dynamically construct components based on data
+- Extend code provided
 
 ## Scaffolding
-The data structure that drives the UI is madlibs/MadLibs.js. To complete your objectives, you will not need to modify this file. The data in this file is an array of Madlib objects. A MadLib object has three properties that you'll utilize:
+The data structure that drives the UI is madlibs/MadLibs.js. To complete your requirements, you will not need to modify this file. The data in this file is an array of Madlib objects. A MadLib object has three properties that you'll utilize:
 - `title` - the title of the Madlib
-- `words` - an array of fill-in-the-blank 'word' objects (these objects contain a `key`, `label`, and `value` property)
-- `getText()` - a function that returns the Madlib text
+- `words` - an array of fill-in-the-blank 'word' objects. Each object contain a `key`, `label`, and `value` property. You will fill in the values
+- `getText()` - a function that returns the Madlib text with each word filled in
 
 ## Requirements
 
 ### Baseline
 As a starting point, review the code that has been provided to you.
 
-To work through the analysis of the existing code, determine how to get the application to load a **random** MadLib, instead of loading the first one which it will currently always do by default.
+To work through the analysis of the existing code, determine how to modify the application to load a **random** MadLib, instead of loading the first one which it currently does by default.
 
 ### Wave 1
-* `WordForm.js` should render text inputs for all word objects in the array - use the `label` property in the word object for the value of the input's [placeholder](http://www.w3schools.com/Tags/att_input_placeholder.asp) attribute.
-  * These text inputs should call the `_onInputChange()` function in `WordForm.js` when a change event occurs - and pass in the appropriate arguments that `_onInputChange()` is expecting.
-* The `Story.js` component should simply display the value returned from the `getText()` function on the `MadLib` object - but only have this component displayed when all word objects have values.
+1. Create a new component which will render the forms inputs for each word in the puzzle.
+  - You should utilize the `selectedMadLib` variable stored in the `App` components `state` to gather the words you need.
+  - You may want to use the `placeholder` HTML attribute to include the `label` property for each word.
+1. Tie each input to an event handler that will modify the state of the `selectedMadLib`.
+  - You will need to pass not only the input's value but also information about _which_ input has changed through this event handler.
+1. Modify the rendering of the `Story` component to only display once all of the input words have been filled out.
```

Comment:
This and the above requirement don't sound like they match up with the controlled forms approach. Aren't we supposed to have them create a single handler for `onSubmit`, rather than updating the state whenever any input changes?